Makefile

```diff
@@ -0,0 +1,55 @@
+CONFIGURATION := Release
+CONSOLE_FRAMEWORK := netcoreapp2.0
+DOLLAR := $$
+DOTNETCORE := 1
+MAXMIND_BENCHMARK_DB := /project/MaxMind.GeoIP2.Benchmark/GeoLite2-City.mmdb
+
+define update_project
+	@docker run \
+		-it \
+		--rm \
+		-v $$(pwd)/$(1):/project \
+		geoip2-dotnet \
+		dotnet outdated -u
+endef
+
+benchmark:
+	@docker run \
+		-it \
+		--rm \
+		-v $$(pwd)/$(1):/project \
+		-e "CONFIGURATION=$(CONFIGURATION)" \
+		-e "CONSOLE_FRAMEWORK=$(CONSOLE_FRAMEWORK)" \
+		-e "DOTNETCORE=$(DOTNETCORE)" \
+		-e "MAXMIND_BENCHMARK_DB=$(MAXMIND_BENCHMARK_DB)" \
+		geoip2-dotnet \
+		/bin/sh -c "dotnet run \
+			-c \$(DOLLAR)CONFIGURATION \
+			-f \$(DOLLAR)CONSOLE_FRAMEWORK \
```

Comment:
Why is `$(DOLLAR)` needed?

Reply:
I had to use it to circumvent the way Make evaluates `$`. 

In this circumstance, both `$CONFIGURATION` and `$CONSOLE_FRAMEWORK` are part of a command that is run within the Docker container, making them strings that require the `$` to be a string. `$` is evaluated by Make. `$$` would bypass the evaluation in Make, but would be evaluated as part of the command. I also attempted to use `$$$`, `$$$$`, `\$$$`, and `\$$$$`, but they were either invalid or evaluated wrongly in the Docker container.  So I ended up assigning `$$` to a variable ensured that Make would perceive `$` as a string when evaluating the `DOLLAR` variable. The preceding backslash was needed for the command to evaluate properly in Docker.

Reply:
Why do you need to circumvent it? With:

```
--- a/Makefile
+++ b/Makefile
@@ -23,9 +23,9 @@ benchmark:
                -e "DOTNETCORE=$(DOTNETCORE)" \
                -e "MAXMIND_BENCHMARK_DB=$(MAXMIND_BENCHMARK_DB)" \
                geoip2-dotnet \
-               /bin/sh -c "dotnet run \
-                       -c \$(DOLLAR)CONFIGURATION \
-                       -f \$(DOLLAR)CONSOLE_FRAMEWORK \
+               /bin/sh -c -x "dotnet run \
+                       -c $(CONFIGURATION) \
+                       -f $(CONSOLE_FRAMEWORK) \
                        -p ./MaxMind.GeoIP2.Benchmark/MaxMind.GeoIP2.Benchmark.csproj"
 
 build-image:
```

I see `+ dotnet run -c Release -f netcoreapp2.0 -p ./MaxMind.GeoIP2.Benchmark/MaxMind.GeoIP2.Benchmark.csproj` being run.

Reply:
Ah, I see. Given that, I can probably avoid passing those two variables as environment variables. I'll clean this up and make another push.